internal/tiltfile2/docker_compose.go

```diff
@@ -0,0 +1,41 @@
+package tiltfile2
+
+import (
+	"fmt"
+
+	"github.com/google/skylark"
+	"github.com/windmilleng/tilt/internal/dockercompose"
+)
+
+// dcResource represents a single docker-compose config file and all its associated services
+type dcResource struct {
+	yamlPath string
+
+	services []dockercompose.Service
+}
+
+func (s *tiltfileState) dockerCompose(thread *skylark.Thread, fn *skylark.Builtin, args skylark.Tuple, kwargs []skylark.Tuple) (skylark.Value, error) {
+	var path skylark.String
```

Comment:
var path string?

it seems like you'd want skylark to do the conversion for you, instead of doing it yourself

Reply:
fair! i think I blindly copypasta'd this from elsewhere in the code.